CallAutomation_CallLiveTranscription/CallAutomationLiveTranscription/Helper.cs

```diff
@@ -0,0 +1,85 @@
+using Azure.Communication.CallAutomation;
+using System.Net.WebSockets;
+using System.Text;
+
+namespace CallAutomation_LiveTranscription
+{
+    public static class Helper
+    {
+        /// <summary>
+        /// Accept WebSocket Connection, and then Loop in receiving data transmitted from client.
+        /// </summary>
+        /// <param name="webSocket"></param>
+        public static async Task ProcessRequest(WebSocket webSocket)
+        {
+            try
+            {
+                while (webSocket.State == WebSocketState.Open || webSocket.State == WebSocketState.CloseSent)
+                {
+                    var buffer = new byte[1024 * 4];
+                    var cancellationToken = new CancellationTokenSource(TimeSpan.FromSeconds(60)).Token;
```

Comment:
60 secs seems less to keep the wesocket to be ideal state. Can you test this by deploying and check if 60 secs provide a good user experience by taking few slightly longer pauses  ?

Reply:
for the quick start it should good, but yes websocket ideal time changes based on the customer use case. For not i will keep it as 60secs